src/components/NewContactButton.tsx

```diff
@@ -0,0 +1,22 @@
+import useWindowSize from '../hooks/useWindowSize';
+
+import '../styles/NewContactButton.scss';
+
+const NewContactButton = () => {
+    const [width] = useWindowSize();
+    const minDeviceWidth = 768;
+    return (
+        <div className="create-contact-btn">
+            <a href="/newContact">
+                {width >= minDeviceWidth ?
+                    <button className="desktop">New contact</button> :
```

Comment:
I would suggest to look into direction of pseudo elements (`:before :after`) Instead of keeping two completely different HTMLs for mobile and desktop you could style it CSS-only and just switch class names of the button
`className={width >= minDeviceWidth ? 'desktop' : 'mobile'}`

It will work this way too, but maybe you'd be interested in another approach.

Reply:
I tried to realize your approach with switching classes depends on width of the screen and using :after selector. Hope I got your idea and thanks for the suggestion)

Reply:
I guess you did get it right)